models/User.go

```diff
@@ -819,25 +827,25 @@ func PlaceAskOrder(userId uint32, ask *Ask) (uint32, error) {
 
 	l.Debugf("Check3: Passed. Creating Ask.")
 
-	if err := createAsk(ask); err != nil {
-		l.Errorf("Error creating the ask %+v", err)
-		return 0, err
-	}
-
-	l.Infof("Created Ask order. AskId: ", ask.Id)
+	oldCash := user.Cash
```

Comment:
commenting here coz can't comment upwards: where is `orderPrice` used in this function? Just curious.